backend/prisma/migrations/20251001090802_change_studentactivity_data_to_json/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - Changed the type of `data` on the `StudentActivityApp` table. No cast exists, the column would be dropped and recreated, which cannot be done if there is data, since the column is required.
+
+*/
```

Comment:
Replace this valid warning (and the warning from the bot) with an explanation why it is okay